packages/trpc/server/routers/viewer/workflows/update.handler.ts

```diff
@@ -309,8 +348,10 @@ export const updateHandler = async ({ ctx, input }: UpdateOptions) => {
                 case WorkflowActions.EMAIL_ATTENDEE:
                   sendTo = bookingInfo.attendees.map((attendee) => attendee.email);
                   break;
-                /*case WorkflowActions.EMAIL_ADDRESS:
-                      sendTo = step.sendTo || "";*/
+                case WorkflowActions.EMAIL_ADDRESS:
+                  await verifyEmailSender(step.sendTo || "", user.id, userWorkflow.teamId, ctx.prisma);
```

Comment:
We are still having the issue that the email doesn't show as verified for a team if a user created it with a verified user email. This `verifyEmailSender` is never called when no reminders for already existing bookings need to be scheduled.


We should call `verifyEmailSender` in a different place. Let's call it when a new step was added and when a step was edited

Reply:
Missed it here, thats why it was never being called when there are no reminders.

https://github.com/calcom/cal.com/blob/f4927a39df89af51d7d108cce1c2e6d1e867efba/packages/trpc/server/routers/viewer/workflows/update.handler.ts#L463-L475

Also throwing error leads to unhandled rejections. Although further steps aren't processed, but the response is shown as 200

![Screenshot 2024-05-17 at 10 37 51 PM](https://github.com/calcom/cal.com/assets/74371312/3494d4d4-9403-4564-b0a2-8e5df7cb87db)